### ISSUE TITLE
[Mod] Headcrab Frenzy 1.5 Compatibility?

### ISSUE THREAD

**Issue by MavisCelus19201** [FWGS/xash3d-fwgs#1900]
I tried running Headcrab Frenzy using the Steam versions 1.5 files, it boots but then just stays on a loading screen without really progressing anywhere. This also doesn't happen with the previous 1.4 version on ModDB though I don't know if it's using VGUI2 or anything especially crazy in this case.

Steam Page: https://store.steampowered.com/app/354900/Headcrab_Frenzy/

**Comment by a1batross** [comment]
I've tried running this mod before, and it worked perfectly for me.

What's your platform?

**Comment by MavisCelus19201** [comment]
Linux, specifically EndeavourOS, after compiling it from source around the 28th of November. Only this mod and Base Defense's latest versions run into issues like it.

**Comment by a1batross** [comment]
Base Defense isn't ported to Linux. If you want to run it, you need to run Xash under Wine.

Headcrab Frenzy has Linux port but check that you downloaded correct depot from Steam.

**Comment by a1batross** [comment]
Yes, you need both client.so and hcfrenzy.so, and xash3d-fwgs must be 32-bit x86 build (`xash3d-fwgs-linux-i386`)

**Comment by a1batross** [comment]
Oh, also, I remember this mod looks for `libsteam_api.so`, it might need to be copied from its original directory to library path (set custom through `LD_LIBRARY_PATH` env var or just where `libxash.so` resides).
